edb/pgsql/delta.py

```diff
@@ -7557,28 +7557,34 @@ def _create_extension(self, ext_spec: str) -> None:
             lclauses.append(f'v.split {op} {pver}')
         cond = ' and '.join(lclauses) if lclauses else 'true'
 
+        ver_regexp = r'^\d+(\.\d+)+$'
```

Comment:
Why do we need this version regexp?

Reply:
Because of this:
```
V2f206a4ba0_main=# select name, version from pg_available_extension_versions where name = 'postgis';
  name   |  version   
---------+------------
 postgis | 3.4.2
 postgis | ANY
 postgis | unpackaged
(3 rows)
```
The version is not necessarily numeric and that causes issues when trying to match the version number by splitting it and converting to ints.